tests/modelgauge_tests/test_pipeline_runner.py

```diff
@@ -266,6 +280,16 @@ def test_unready_ensemble(self, tmp_path, prompts_dataset, annotators):
                 output_dir=tmp_path,
             )
 
+    def test_ready_ensemble(self, tmp_path, prompts_dataset, safety_annotators):
```

Comment:
This is a very mysterious test. I'm wondering if running the readiness checks in __init__ is the best way to do this. Perhaps the `ensure_ready` should be directly callable and normally called by the things that call `run`? But feel free to merge this to fix your thing.